ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -12328,6 +12549,14 @@ IHqlExpression * HqlTreeNormalizer::createTransformedBody(IHqlExpression * expr)
     case no_record:
         {
             OwnedHqlExpr transformed = completeTransform(expr);
+            if (transformed->hasAttribute(lookupAtom))
+            {
+                StringBuffer filename;
+                IHqlExpression *dfsAttr = transformed->queryAttribute(lookupAtom);
+                if (dfsAttr)
+                    dfsAttr->queryChild(0)->queryValue()->getStringValue(filename);
```

Comment:
I think I know that it is constant (I create this attribute for the purpose of outputting this error message...)